### PR TITLE
[SecurityBundle] Deprecate the `enable_authenticator_manager` option

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | 6.2
| Bug fix?      | no
| New feature?  | no
| Deprecations? | yes
| Tickets       | -
| License       | MIT
| Doc PR        | -

Last step before we close the chapter in 7.0 🎉

### CODE REVIEW

File: src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/Fixtures/php/listener_provider.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?php
 
 $container->loadFromExtension('security', [
-    'enable_authenticator_manager' => true,
+
```
Comment (alamirault) [inline src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/Fixtures/php/listener_provider.php]: We can remove the blank line

File: src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/Fixtures/xml/access_decision_manager_customized_config.xml [hunk 2]
```diff
@@ -7,7 +7,7 @@
         http://symfony.com/schema/dic/security
         https://symfony.com/schema/dic/security/security-1.0.xsd">
 
-    <config enable-authenticator-manager="true">
+    <config >
```
Comment (alamirault) [inline src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/Fixtures/xml/access_decision_manager_customized_config.xml]: I think we must remove extra space in all XMLs

Comment (chalasr) [reply src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/Fixtures/xml/access_decision_manager_customized_config.xml]: fixed, thanks

**Review by welcoMattic — APPROVED** [review]

**Review by alamirault — APPROVED** [review]

**Review by derrabus — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @chalasr.

**Pull request merged**